src/FieldWarning/Assets/Units/Component/Armor/ArmorComponent.cs

```diff
@@ -0,0 +1,175 @@
+﻿/**
+ * Copyright (c) 2017-present, PFW Contributors.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License"); you may not use this file except in
+ * compliance with the License. You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software distributed under the License is
+ * distributed on an "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See
+ * the License for the specific language governing permissions and limitations under the License.
+ */
+
+using System.Collections;
+using System.Collections.Generic;
+using UnityEngine;
+using PFW.Units.Component.Movement;
+using PFW.Units.Component.Damage;
+using PFW.Units.Component.Weapon;
+using PFW.Units.Component.Health;
+
+
+namespace PFW.Units.Component.Armor
+{
+    class ArmorComponent
+    {
+        /// <summary>
+        /// The movement component where we obtain the front and right vectors
+        /// used in deciding which face of the armor is hit
+        /// </summary>
+        public MovementComponent Unit { get; private set; }
+        private PlatoonBehaviour _platoon;
+        private HealthComponent _healthComponent;
```

Comment:
Unity warning: 

> _healthComponent is never assigned to and will always have its default value null

It can be removed by typing `_healthComponent = null` if intentional but I guess it is not, because you do use it in handleHit().
.